reporter-web-app/src/components/PackageFindingsTable.js

```diff
@@ -206,7 +207,9 @@ const PackageFindingsTable = (props) => {
             key: 'path',
             sorter: (a, b) => a.path.length - b.path.length,
             textWrap: 'word-break',
-            title: 'Path'
+            title: 'Path',
+            // Search bar
+            ...getColumnSearchProps('path', {}, webAppPackage)
```

Comment:
This only adds search but without the state management so this will throw errors in Ant Design as getColumnSearchProps interferes with unmanaged state of PackageFindingsTable.

Reply:
I'm not sure about what errors you are talking about and how to fix them. Would you mind explaining?

Reply:
I need more guidance to address this issue